### PR TITLE
Enable dependabot auto-merge

### PR DESCRIPTION
**Pull request by LucWollants**
### Changed
- Enable dependabot auto-merge, see https://github.com/dependabot/fetch-metadata?tab=readme-ov-file#enabling-auto-merge

### Note
- Extra dependabot secret `DEPENDABOT_AUTO_MERGE_PERSONAL_ACCESS_TOKEN` was created, see https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic

### CODE REVIEW

**Review by JonasVHG — APPROVED** [review]

File: .github/workflows/dependabot-automerge.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.user.login == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1
+      - name: Enable auto-merge for Dependabot PRs
+        if: ${{contains(steps.dependabot-metadata.outputs.dependency-names, 'rails') && steps.dependabot-metadata.outputs.update-type == 'version-update:semver-patch'}}
```
Comment (simon-debruijn) [inline .github/workflows/dependabot-automerge.yml:15]: `rails` ?

**Review by simon-debruijn — APPROVED** [review]
Could potentially cause issues. Since it sometimes opens separate PR for dependencies that should be updated at the same time. But we can try it out 👍🏻 

e.g.
react and react-dom libs

File: .github/workflows/dependabot-automerge.yml (hunk 1 above)
Comment (LucWollants) [reply .github/workflows/dependabot-automerge.yml]: Nicely spotted. I took another approach and used https://github.com/daneden/enable-automerge-action

**Comment by LucWollants** [comment]
> Could potentially cause issues. Since it sometimes opens separate PR for dependencies that should be updated at the same time. But we can try it out 👍🏻
> 
> e.g. react and react-dom libs

But it will only merge when the GitHub actions are successful and it will open new pull requests when one is merged to the main branch.

**Comment by simon-debruijn** [comment]
> > Could potentially cause issues. Since it sometimes opens separate PR for dependencies that should be updated at the same time. But we can try it out 👍🏻
> > e.g. react and react-dom libs
> 
> But it will only merge when the GitHub actions are successful and it will open new pull requests when one is merged to the main branch.

I'm afraid github actions won't catch a mismatch in react libs. Or a mismatch in eslint libs (maybe)

**Pull request merged**